Gemfile

```diff
@@ -7,14 +7,14 @@ ruby "3.0.2"
 
 gem "awesome_print"
 gem "bootsnap", ">= 1.4.4", require: false
-gem "cancancan"
 gem "devise"
 gem "devise-i18n"
 gem "jquery-rails"
 gem "jsonapi-serializer"
 gem "pg", "~> 1.0"
 gem "pry-rails"
 gem "puma", "~> 5.4"
+gem "pundit"
```

Comment:
this does not belong in this PR 🤔

Reply:
You're right. 
I don't know how I merged the two